test/test_test.jl

```diff
@@ -2,6 +2,8 @@ include("common.jl")
 
 MPI.Init()
 
+@test MPI.Get_processor_name() == gethostname()
```

Comment:
Note that documentation of [`MPI_Get_processor_name`](https://www.open-mpi.org/doc/v3.1/man3/MPI_Get_library_version.3.php) specifically says the function doesn't necessarily return the same as `gethostname`, but honestly I wouldn't know what to test otherwise, apart from `MPI.Get_processor_name() isa String`?